drivers/npmx/npmx_shell.c

```diff
@@ -2706,6 +2706,75 @@ static int cmd_vbusin_status_connected_get(const struct shell *shell, size_t arg
 	return 0;
 }
 
+static int cmd_vbusin_current_limit_get(const struct shell *shell, size_t argc, char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
+
+	if (npmx_instance == NULL) {
+		shell_error(shell, "Error: shell is not initialized.");
+		return 0;
+	}
+
+	npmx_vbusin_current_t current_limit;
+	npmx_vbusin_t *vbusin_instance = npmx_vbusin_get(npmx_instance, 0);
+	npmx_error_t err_code = npmx_vbusin_current_limit_get(vbusin_instance, &current_limit);
+
+	uint32_t current;
+	npmx_vbusin_current_convert_to_ma(current_limit, &current);
```

Comment:
Return value should be checked here.

Reply:
It is a PMIC value, so it always is a proper value. Checking the return value can be skipped.

Reply:
Discussed with @masz-nordic that the result should be checked.

Reply:
Checking added